.gitignore

```diff
@@ -1,6 +1,10 @@
 node_modules/*
 Bloodhound-*
 dist/
+<<<<<<< HEAD
```

Comment:
just want to note that multiple merge markers are committed in this pullrequest.